README.md

```diff
@@ -1,70 +1,80 @@
-# Getting Started with Create React App
+# Todo-List App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+"Todo-List App" is a web application that allows you to organize your tasks. 
 
-## Available Scripts
+The present project is part of the Microverse curriculum Module 3 Block 2. The main objective is putting hands on practice to build a web application using React.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- Html/CSS
+- JavaScript
+- React
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Additional Tools
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- Webhint
+- Stylelint
+- Eslint
+- Lighthouse
+- Webpack
 
-### `npm test`
+## Live Demo 
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[Live Demo App on Github-Pages](https://carshy.github.io/todos-react-application/)
```

Comment:
- [ ] While you did a good job by including a live demo your program is running perfectly on my local machine, but in the live demo, I am not able to add any todo (the add button is not functional). please kindly fix that problem.